src/Compilers/CSharp/Portable/Compiler/MethodBodySynthesizer.cs

```diff
@@ -134,7 +134,7 @@ internal static class MethodBodySynthesizer
 
             foreach (var field in synthesizedFields.FieldSymbols)
             {
-                var targetScriptType = (ImplicitNamedTypeSymbol)field.Type.TypeSymbol;
+                var targetScriptType = field.Type.TypeSymbol;
```

Comment:
>var [](start = 16, length = 3)

Please spell out the type

Reply:
Reverted change.

---
In reply to: [190711632](https://github.com/dotnet/roslyn/pull/26356#discussion_r190711632) [](ancestors = 190711632)